engine/flink/schemed-kafka-components-utils/src/test/scala/pl/touk/nussknacker/engine/schemedkafka/AvroNodesClassloadingSpec.scala

```diff
@@ -1,63 +0,0 @@
-package pl.touk.nussknacker.engine.schemedkafka
-
-import cats.effect.SyncIO
-import cats.effect.kernel.Resource
-import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient
-import io.confluent.kafka.schemaregistry.testutil.MockSchemaRegistry
-import org.scalatest.funsuite.AnyFunSuite
-import org.scalatest.matchers.should.Matchers
-import pl.touk.nussknacker.engine.api.ProcessVersion
-import pl.touk.nussknacker.engine.api.definition.EngineScenarioCompilationDependencies
-import pl.touk.nussknacker.engine.build.ScenarioBuilder
-import pl.touk.nussknacker.engine.definition.test.{ModelDataTestInfoProvider, TestingCapabilities}
-import pl.touk.nussknacker.engine.definition.test.TestInfoProvider.TestingCapabilitiesError.SourceCompilationError
-import pl.touk.nussknacker.engine.process.helpers.TestResultsHolder
-import pl.touk.nussknacker.engine.schemedkafka.AvroNodesClassloadingSpec.sinkForInputMetaResultsHolder
-import pl.touk.nussknacker.engine.schemedkafka.KafkaAvroIntegrationMockSchemaRegistry.schemaRegistryMockClient
-import pl.touk.nussknacker.engine.schemedkafka.helpers.SchemaRegistryMixin
-import pl.touk.nussknacker.engine.schemedkafka.schemaregistry.SchemaVersionOption
-import pl.touk.nussknacker.engine.schemedkafka.schemaregistry.universal.MockSchemaRegistryClientFactory
-import pl.touk.nussknacker.engine.spel.SpelExtension._
-import pl.touk.nussknacker.engine.testing.LocalModelData
-import pl.touk.nussknacker.engine.util.ThreadUtils
-import pl.touk.nussknacker.test.FailingContextClassloader
-
-class AvroNodesClassloadingSpec extends AnyFunSuite with Matchers with SchemaRegistryMixin {
```

Comment:
This is the only test that was based on these internals. I recommend to remove it and revert it as some higher level test when we find it important